NanoProd/crab/Run3_2022EE/DY.yaml

```diff
@@ -3,7 +3,7 @@ config:
     sampleType: mc
     era: Run3_2022EE
 
-DYJetsToLL_M-50: /DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v5-v2/MINIAODSIM
+DYJetsToLL_M-50-madgraphMLM: /DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v5-v2/MINIAODSIM
```

Comment:
wrong GT => remove

Reply:
`Run3Summer22EEMiniAODv4`

Reply:
This has been resolved and exceptions have been updated.